target/linux/ath25/image/Makefile

```diff
@@ -73,32 +73,37 @@ endef
 TARGET_DEVICES += generic
 
 define Device/ubnt2-pico2
-  DEVICE_TITLE := Ubiquiti XS2-8
+  DEVICE_VENDOR := Ubiquiti
+  DEVICE_MODEL := XS2-8
   IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | pad-to 128k | mkfwimage XS2-8 -v XS2.ar2316
 endef
 TARGET_DEVICES += ubnt2-pico2
 
 define Device/ubnt2
-  DEVICE_TITLE := Ubiquiti XS2
+  DEVICE_VENDOR := Ubiquiti
+  DEVICE_MODEL := XS2
   IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | pad-to 128k | mkfwimage XS2 -v XS2.ar2316
 endef
 TARGET_DEVICES += ubnt2
 
 define Device/ubnt5
-  DEVICE_TITLE := Ubiquiti XS5
+  DEVICE_VENDOR := Ubiquiti
+  DEVICE_MODEL := XS5
   IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | pad-to 128k | mkfwimage XS5 -v XS5.ar2313
 endef
 TARGET_DEVICES += ubnt5
 
 define Device/np25g
-  DEVICE_TITLE := np25g
+  DEVICE_VENDOR := Compex
+  DEVICE_MODEL := np25g
   KERNEL := kernel-bin | gzip-kernel
   IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | pad-to 128k | mkmylofw np25g
 endef
 #TARGET_DEVICES += np25g
 
 define Device/wpe53g
-  DEVICE_TITLE := wpe53g
+  DEVICE_VENDOR := Compex
+  DEVICE_MODEL := wpe53g
```

Comment:
I think this should be capital letters.

Reply:
Thanks for the review!

Reply:
I was referring to the DEVICE_MODEL.

Reply:
Ok, I searched compex and saw that is was written upper case on some sites.